examples/logical_interconnect_group.rb

```diff
@@ -23,19 +23,18 @@
 # Simple Logical Interconnect group creation without interconnects and uplinks
 lig_data = {
   enclosureType: 'C7000',
-  type: 'logical-interconnect-groupV3'
 }
```

Comment:
I like this example better than before :) There's just a few things I noticed or would suggest:
 - It doesn't look like the `lig_data` object is ever used. It probably just got left in on accident.
 - On many of our examples we name our fake client object "client". This may cause some confusion for people when they see the resource that contains `client client`. I'm going to try to name the client object `my_client` in the future to avoid this confusion. I'd suggest it here too.
 - Lastly, for the resources where we're using the default action, I'd recommend leaving it out altogether (default is :create for this resource). When we leave it out, we'd obviously want to have a comment in there somewhere saying that it's the default, and that's why we left it out. This one is not very important because it works with or without it, but it may simplify some of the examples.

And in general, the more comments in these examples, the better :smile: